app/controllers/v0/in_progress_forms_controller.rb

```diff
@@ -32,5 +33,16 @@ def destroy
       form.destroy
       render json: form
     end
+
+    def proxy_add
+      if params[:id] == '21-526EZ'
+        if @current_user.participant_id.nil?
+          # Add person to MVI if missing participant_id (no matter if birls_id present or absent)
+          @current_user.mvi.mvi_add_person
+        elsif @current_user.birls_id.nil?
+          raise Common::Exceptions::ValidationErrors, 'No birls_id while participant_id present'
```

Comment:
It may be worth our time adding in some logging here. This is an extreme edge case (as it *technically* should never be possible to have a corp id but not a birls id) and we may want to log some user information so we can investigate these cases with upstream services.

Reply:
I agree that we should log this error. My reading of this controller is that it inherits from ApplicationController and the application controller will log the error to sentry since `Common::Exceptions::ValidationErrors` isn't in the `SKIP_SENTRY_EXCEPTION_TYPES` array. Am I reading this? ref:https://github.com/department-of-veterans-affairs/vets-api/blob/4e72f8bf232efbd02ec9d2945b261ca35ecb69cf/app/controllers/application_controller.rb#L109

Reply:
I think you are correct and looking deeper into that it looks like `log_exception_to_sentry` will also log it to the rails console, so i think we are good there.